frontend-react/src/components/AuthElement.test.tsx

```diff
@@ -5,15 +5,19 @@ import { mockSessionContext } from "../contexts/__mocks__/SessionContext";
 import { mockFeatureFlagContext } from "../contexts/__mocks__/FeatureFlagContext";
 import { FeatureFlagName } from "../pages/misc/FeatureFlags";
 import { RSSessionContext } from "../contexts/SessionContext";
-import { renderWithBase } from "../utils/CustomRenderUtils";
+import { renderApp } from "../utils/CustomRenderUtils";
 
 import { AuthElement } from "./AuthElement";
 
 const mockUseNavigate = jest.fn();
 
-jest.mock("react-router", () => ({
-    useNavigate: () => mockUseNavigate,
-}));
+jest.mock("react-router", () => {
```

Comment:
Smallest point in the world, but I think we could simplify this even further, no?

```tsx
jest.mock("react-router", () => ({
    ...jest.requireActual("react-router"),
    useNavigate: () => mockUseNavigate
}));
```